IBM/MAX-Object-Detector
# Test Other Endpoints

- test other endpoints
- increase travis sleep time from 10 to 30 secs just to be safe
- show download file progress
- install from requirements.txt (resolves #23)
- verify md5 hashes of downloaded files
- update the swagger screenshot (resolves #27)
